cli/flags.rs

```diff
@@ -208,7 +210,23 @@ impl Default for CheckFlag {
 }
 
 #[derive(Clone, Debug, PartialEq, Default)]
-pub struct Flags {
+pub struct Flags(pub(crate) Arc<Inner>);
```

Comment:
I think that we don't need to change this interface? Probably we only need to wrap it in an `Arc` in a couple places downstream from this file and maybe put it in an `Arc` before it goes into the proc state. Also, it seems like we're just needlessly cloning `Flags` in a few places because we could just use `ps.flags` instead.

Reply:
I agree, no changes should be necessary to `Flags` struct itself

Reply:
@ry I believe we'd only need to wrap it in an `Arc` when putting it in `ProcState` and then change some of the other API to not requiring cloning (ex: https://github.com/denoland/deno/pull/13610#discussion_r800233365)? Then if some other place needs that state they could do `let flags = ps.flags.clone();`?

Reply:
Ok sure - but we should remove the `derive(Clone)` from the Flags struct then to prevent people from copying this.

Reply:
@dsherret @bartlomieju @ry 
actually I did it. Arc only in ProcState